targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_MIMXRT1170/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2020-2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+add_subdirectory(TARGET_EVK EXCLUDE_FROM_ALL)
+
+if(${MBED_TOOLCHAIN} STREQUAL "GCC_ARM")
+    set(STARTUP_FILE device/TOOLCHAIN_GCC_ARM/startup_MIMXRT1176_cm7.S)
+    set(LINKER_FILE device/TOOLCHAIN_GCC_ARM/MIMXRT1170xxxxx.ld)
+endif()
+
+add_library(mbed-mimxrt1170-evk INTERFACE)
+
+target_include_directories(mbed-mimxrt1170-evk
+    INTERFACE
+        .
+        device
+        drivers
+)
+
+target_sources(mbed-mimxrt1170-evk
+    INTERFACE
+     
+        device/system_MIMXRT1176_cm7.c
+
+        drivers/fsl_anatop.c
+        drivers/fsl_anatop_ai.c
+        drivers/fsl_cache.c
+        drivers/fsl_clock.c
+        drivers/fsl_common.c
+        drivers/fsl_dcdc.c
+        drivers/fsl_flexspi.c
+        drivers/fsl_flexspi_nor_boot.c
+        drivers/fsl_gpio.c
+        drivers/fsl_lpi2c.c
+        drivers/fsl_lpspi.c
+	    drivers/fsl_lpuart.c
```

Comment:
can you fix alignment in this file

Reply:
i moved it, but on my end both of these seem aligned.

Reply:
Are you using tabs instead of spaces in your editor?

Reply:
to be honest i dont recall I think I use a mix of both depending on how far in the alignment is.... if I open the fil right now I see it aligned, however if I open it online I can see this issue present..

Reply:
actually, if I click edit online the alignment is fine

Reply:
Please ensure you use spaces instead of tabs to have correct indentations everywhere.
Thanks

Reply:
ok fixed